### PR TITLE
Minor py fixes

### PR DESCRIPTION
**Pull request by joshmoore**
Propose to get this out as 6.0.0a2 today.

cc: @sbesson

### CODE REVIEW

File: ome_model/experimental.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def parse_tiff(tiff):
     return (m.group("channel"), m.group("time"), m.group("slice"))
 
 
-def create_companion(plates=[], images=[]):
+def create_companion(plates=[], images=[], out=sys.stdout):
```
[the remark this answers is not in the document]
Comment (joshmoore) [reply ome_model/experimental.py]: Do you have a short-term preference?

File: ome_model/experimental.py [hunk 2]
```diff
@@ -6,11 +6,11 @@
 import uuid
 import xml.etree.ElementTree as ET
 
-
 OME_ATTRIBUTES = {
+    'Creator': "ome_model/experimental.py",
```
[the remark this answers is not in the document]
Comment (joshmoore) [reply ome_model/experimental.py]: I definitely thought about that, but it leads to needing `ome_model.VERSION` etc. Propose to handle that in a follow-up.

File: ome_model/experimental.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sbesson) [reply ome_model/experimental.py]: Slight preference towards the latter option as inspired by your proposed change i.e. reduce the scope of this method to generating a string representation of the metadata and leave downstream code free to choose how to consume it.

[the remark this answers is not in the document]
Comment (joshmoore) [reply ome_model/experimental.py]: Strike that. Getting the `<? xml` header via `tostring` is a no go. I'm going to say let's keep passing in the output stream for the moment. See the last commit.

**Pull request merged**